### DISCUSSION TITLE
Add Pin Tab Keyboard Shortcut

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#793] [Ideas]
### What feature would you like to see?

A feature which Arc nails is the pinned tabs one. We are 99% there in Zen - but we need a keyboard shortcut to Pin a Tab.

There are extension like this one which do the job:

https://addons.mozilla.org/en-US/firefox/addon/toggle-pin-tab/

Hope it's integrated in Zen soon

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/577_

**Comment by filiptrplan** [comment]
This would be great to have. Also for the new essentials feature. Are the maintainers considering this?

**Comment by negativems** [comment]
Would be great! also for adding to essentials.

**Comment by yashcrest** [comment]
is this still only possible through a extension? Is there still no native way to have a shortcut to pin a tab in Zen browser? I am still so much hesitant to switch to Zen browser for these missing functionality. I use these features on arc heavily.

**Comment by apotenza92** [comment]
Yes, a shortcut option to pin or add to essentials, and have the same shortcut unpin or remove from essentials if the current tab has been added already.

**Comment by timurmaio** [comment]
The essential missing piece from Arc!

**Comment by naokichau** [comment]
+1 here

**Comment by tomirodriguez** [comment]
needed ! +1

**Comment by AurelienJt** [comment]
Yes please!

**Comment by snarum** [comment]
I'm giving Zen another shot today, but I quickly discover that there is not shortcut for this. Should be easy to introduce, so the fact that it is missing makes me wonder if I misunderstand the purpose of pinned tabs. In Arc i used ctrl-D all the time.

**Comment by koichiio** [comment]
For now, this add-on allows us to do that. https://addons.mozilla.org/en-US/firefox/addon/pinunpin-tab/

**Comment by mr-cheffy** [comment]
One already exists now! Closing.

**Reply by inglar** [reply]
where is it? I don't see any in https://docs.zen-browser.app/user-manual/shortcuts

**Reply by Ivaon** [reply]
@inglar It’s the "Toggle Pin tab" one:
 
<img width="975" height="517" alt="image" src="https://github.com/user-attachments/assets/1864399e-ab3c-41c7-8dab-43e153b7b41e" />

**Reply by inglar** [reply]
@Ivaon, thanks!
`Cmd-Shift-D` on mac
